model/configs/default_pcfg.py

```diff
@@ -41,24 +41,25 @@ def get_config():
   config.per_device_batch_size = 64
 
   # Beam size for inference.
-  config.beam_size = 4
+  config.beam_size = 1
 
-  config.num_train_steps = 10_000
+  config.num_train_steps = 33_000
 
   # Number of steps to take during evaluation.
-  config.num_eval_steps = 1
+  config.num_eval_steps = 5
   # Number of steps to generate predictions.
   # -1 will use the whole eval dataset.
-  config.num_predict_steps = -1
+  config.num_predict_steps = 1
```

Comment:
Just checking if we want to keep `1` as the default.  (Same for num_predict_loops.)

Reply:
I'll do -1 for num_predict_steps and 1 for num_predict_loops when use_annottions = True.